neo/rawio/spikegadgetsrawio.py

```diff
@@ -104,6 +123,20 @@ def _parse_header(self):
         self._sampling_rate = float(hconf.attrib["samplingRate"])
         num_ephy_channels = int(hconf.attrib["numChannels"])
 
+        # check for agreement with number of channels in xml
+        sconf_channels = np.sum([len(x) for x in sconf])
+        if sconf_channels < num_ephy_channels:
+            num_ephy_channels = sconf_channels
+        if sconf_channels > num_ephy_channels:
```

Comment:
Sorry to interrupt, but is that necessarry to add a comparing that num_chan_per_chip > num_ephy_channels?
I got a recording recently and found that num_chan_per_chip is ocasionally read as `339134394`.
I manage to extract the xml file and see
` <SpikeConfiguration categories="" chanPerChip="339134394" device="intan">
`
which implies some error...

Reply:
do you mean the workspace file you use to collect data has `chanPerChip="339134394"`?

Reply:
No, we are using the workspace file with `chanPerChip="32"`, but when we check the output `.rec` file, it becomes `chanPerChip="339134394"`. We're still trying to figure out what's going on...